packages/azure-services/src/azure-queue/queue.spec.ts

```diff
@@ -233,12 +241,17 @@ describe(Queue, () => {
 
         test.each([null, { messageId: null }])('creates message failed - response = %o', async response => {
             const messageText = 'some message';
+            let caughtError: Error;
 
+            setupRetryHelperMock();
             setupQueueCreationCallWhenQueueExists();
             setupVerifyCallToEnqueueMessage(messagesURLMock, messageText, response);
 
-            const isCreated = await testSubject.createMessage(queue, messageText);
-            expect(isCreated).toBe(false);
+            await testSubject.createMessage(queue, messageText).catch(reason => {
+                caughtError = reason;
+            });
+
+            expect(caughtError.message).toBe(`Failed to enqueue the message`);
 
```

Comment:
instead you could do, 
```suggestion
await expect(testSubject.createMessage(queue, messageText)).rejects.to(new Error(`Failed to enqueue the message');
```